docs/extending-documentation.md

```diff
@@ -1,119 +1,169 @@
 # Extending Lion Documentation
 
-```js script
-export default {
-  title: 'Guidelines/Extending documentation',
-};
-```
-
-If you extend [Lion](https://lion-web-components.netlify.app/) components, you don't only want to reuse the components, but you probably want to reuse the documentation (Storybook demos) as well. Wouldn't it be nice to just take it all from lion, but replace it with your own design system extension, so you don't have the extra maintenance of essentially copying the docs from `Lion` for your own design system implementation?
+(rewrite of original)
+TODO: Check medium support for IDs in MarkDown
+TODO: Add IDs
+TODO: Make TOC
+TODO: Consistent person
 
-In this blog we will explain how `Lion` supports this use case, and allows you to extend not just the components, but also the documentation.
+Lion is a set of white label cross-platform Web Components you can use as the base for your own design system. You can extend them with your own styling for an accessible, performant and complete design system. 
```

Comment:
I would also adjust this second sentence:
- it now looks a bit like we get a11y and perf as a a consequence of adding styles :) 
- reads a bit nicer if we don't end two consecutive sentences with `your own design system`